src/api/encoder.py

```diff
@@ -0,0 +1,37 @@
+"""Encoding Utility Logic."""
+
+# Third-Party Libraries
+from connexion.apps.flask_app import FlaskJSONEncoder  # type: ignore
+import six  # type: ignore
+
+# cisagov Libraries
+from api.models.base_model_ import Model
+
+
+class JSONEncoder(FlaskJSONEncoder):
+    """Encoding Class."""
+
+    include_nulls = False
+    encoding_type = "UTF-8"
```

Comment:
Alphabetize wherever possible:
```suggestion
    encoding_type = "UTF-8"
    include_nulls = False
```